### DISCUSSION TITLE
Request: PyQtGraph maintainers include screenshots of argos at SciPy Poster Board Session

### DISCUSSION THREAD

**Discussion by j9ac9k** [titusjan/argos#28] [General]
Hi @titusjan 

Sorry for the noise on your discussions/issue tracker, I wasn't sure how to reach out to you otherwise.

We (pyqtgraph) have a poster board session at the SciPy conference this year.  A large part of our poster board will include screenshots of PyQtGraph being used "in the wild" doing "real work".   We would like part of the collage we create to include a screenshot or two of argos.  We would of course properly attribute your project.

If you'd like to continue this conversation in private, I have contact info on my github profile.

Thanks!
Ogi

**Comment by titusjan** [comment]
Hi Ogi.

I am flattered that you want to include Argos on your poster, so of course you have my permission. I will send you an email so that you have my contact info as well.

Pepijn.

**Reply by j9ac9k** [reply]
https://drive.google.com/file/d/1-Q6Akm5nbIbu-3KeJyPR2PKFUr1_LyOo/view?usp=sharing

We sent this to the printers late last night, thanks for allowing us to use argos is part of our collage!

**Reply by titusjan** [reply]
Very nice poster! Good luck at the conference.
